wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -35,6 +35,7 @@
             wagtailConfig.LOCALES = {{ locales|safe }};
 
             wagtailConfig.STRINGS = {% js_translation_strings %};
+            wagtailConfig.CSRF_TOKEN = '{{ csrf_token }}'
```

Comment:
Has this been included accidentally? I can't see its use here.

Reply:
Oops, yes it has